src/main/java/seedu/allonus/modules/ModuleParser.java

```diff
@@ -360,27 +371,57 @@ public String validateModuleTime(String time) throws ModuleTimeException {
         String timeSlot = null;
         if (matcher.find()) {
             timeSlot = matcher.group().trim();
+            timeSlot = checkTimeStartEarlier(timeSlot);
             return timeSlot;
         } else {
             throw new ModuleTimeException(WRONG_TIME_FORMAT_MESSAGE);
         }
     }
 
+    // Checks if given start time is earlier than given end time in a String timeSlot
```

Comment:
Perhaps this should be in Javadoc format instead?

Reply:
Since it was a private method I did not add as a javadoc format. Wrote a normal comment to the reader instead!